AlphaWallet/Tokens/Types/NonFungibleFromJson.swift

```diff
@@ -22,6 +22,17 @@ protocol NonFungibleFromJson: Codable {
     var generationTrait: OpenSeaNonFungibleTrait? { get }
     var collectionCreatedDate: Date? { get }
     var collectionDescription: String? { get }
+    var meltValue: String? { get }
+    var meltFeeRatio: Int? { get }
+    var meltFeeMaxRatio: Int? { get }
+    var totalSupply: String? { get }
```

Comment:
I suppose all the new properties are optional, so it doesn't affect JSON strings that we stored in Realm earlier, right?

Reply:
not sure. it does, because we update existing json with additional information contained in these fields. (ps if i understood u right)